uwp-project/devilutionx.vcxproj.filters

```diff
@@ -0,0 +1,393 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Does this define which files to package with the app bundle? Ideally, in the future, we'd generate this file via CMake to not have to maintain the list of assets in 2 places.

Reply:
`.filter` only organize them when the project is opened on visual studio. the actual list of files is in the `.vcxproj` file

Reply:
Was this generated in some way? Normally we generate the projects via CMake, (even for Android the file list is picked up from there).

Reply:
i have no idea how to do it with cmake. i'll have to check

Reply:
What I suggest is doing something like the way I have changed the app version in the manifest using PowerShell. I'll do it later and push.

Reply:
Recent versions of Visual Studio has CMake support build in where you can simple open the CMakeLists.txt file as the project file. But you can also generate a sln file via CMake:
`cmake -G "Visual Studio 17 2022" .` Not sure if that is what you where asking :)

Reply:
I thought you were asking about the assets :) no it wasn't generated. I created the solution on visual studio and configured it by hand.

Reply:
there is a way of adding files to the project using wildcards. so i changed the `vcxproj` file for looking into `build/assets` and linking all the files there. this way any changes on assets will be reflected in the build.

Reply:
Looks a whole lot better now :)